spec/adapters/googlecharts/base_chart_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper.rb'
+
+describe GoogleVisualr::BaseChart do
+  before { Daru::View.plotting_library = :googlecharts }
+  before(:each) do
+    @query_string = 'SELECT A, H, O, Q, R, U LIMIT 5 OFFSET 8'
```

Comment:
We must use `let` instead of `@`. 
Please follow [better specs](http://www.betterspecs.org/).